layers/meta-balena-allwinner/recipes-bsp/u-boot/u-boot_2018.09.bbappend

```diff
@@ -1,17 +1,13 @@
 UBOOT_KCONFIG_SUPPORT = "1"
 inherit resin-u-boot
-
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
 
 #remove the resin-specific-env-integration-kconfig.patch patch from meta-resin
-#and the 0001-nanopi_neo_air_defconfig-Enable-eMMC-support.patch patch from
-#meta-sunxi because these fail to apply
-SRC_URI_remove = "file://0001-nanopi_neo_air_defconfig-Enable-eMMC-support.patch \
-                  file://resin-specific-env-integration-kconfig.patch"
+#meta-sunxi because it fails to apply
+SRC_URI_remove = "file://resin-specific-env-integration-kconfig.patch"
 
-#Add updated patches for 0001-nanopi_neo_air_defconfig-Enable-eMMC-support.patch
-#and for 0001-nanopi_neo_air_defconfig-Enable-eMMC-support.patch
-SRC_URI += "file://0001-nanopi_neo_air_defconfig-Enable-eMMC-support.patch \
-            file://0001-Add-Resin-specific-boot-command.patch \
-            file://resin-specific-env-integration-kconfig.patch \
-           "
+#Add updated patch for resin-specific-env-integration-kconfig_redone.patch
```

Comment:
This needs rewording again.
`Add updated patch for resin-specific-env-integration-kconfig_redone.patch`
I don't understand what adding a patch for a patch means.

Reply:
Did the re-wording.